ecs/task_definition_PROD.json

```diff
@@ -0,0 +1,186 @@
+{
+  "containerDefinitions": [
+    {
+      "name": "fee-oracle-container-PROD",
+      "essential": true,
+      "image": "852551629426.dkr.ecr.us-east-2.amazonaws.com/fee-oracle:stable",
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "/ecs/fee-oracle-PROD",
+          "awslogs-region": "us-east-2",
+          "awslogs-stream-prefix": "ecs"
+        }
+      },
+      "portMappings": [
+        {
+          "containerPort": 8091,
+          "hostPort": 8091,
+          "protocol": "tcp"
+        }
+      ],
+      "environment": [
+        {
+          "name": "DD_SERVICE",
+          "value": "fee-oracle"
+        },
+        {
+          "name": "APP_MODE",
+          "value": "release"
+        },
+        {
+          "name": "WORKING_ENV",
+          "value": "dev"
```

Comment:
will this file be used for PROD? or testnet? if for PROD, this value should be `production` and the `HTTP_SERVER_MODE` should be `release`

Reply:
@freddyli7 do you know what are the values for testnet?